pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -43,12 +43,21 @@ template <class StateVectorT, class Derived> class MeasurementsBase {
     using PrecisionT = typename StateVectorT::PrecisionT;
     using ComplexT = typename StateVectorT::ComplexT;
 
+#ifdef _ENABLE_PLGPU
+  protected:
+    StateVectorT &_statevector;
```

Comment:
Is there a reason this cannot be const in this instance? How does LK handle this?

Reply:
Aha, nice spot! I'm about to try if `const` work with `cusparseCreateConstDnVec`. `const` doesn't work for the moment because the dense vector to be passed to SparseMV API is created by `cusparseCreateDnVec` now. `cusparseCreateDnVec` only works for non const pointer. Will keep you posted once I tried the `cusparseCreateConstDnVec`.

Reply:
See above.

Reply:
Done!